cloudant-client/src/test/java/com/cloudant/tests/QueryTests.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2017, 2018 IBM Corp. All rights reserved.
+ * Copyright © 2017, 2018, 2020 IBM Corp. All rights reserved.
```

Comment:
Nit: I don't think the "middle" year is necessary.
```suggestion
 * Copyright © 2017, 2020 IBM Corp. All rights reserved.
```